packages/api-browser/window.js

```diff
@@ -23,4 +33,14 @@ const objectToFeaturesString = (features) => {
   }).join(',');
 };
 
+// if we have an opener, we are not the parent so we need to add it as a window
+if (window.opener) {
```

Comment:
I could move this into `app.js` to be part of the bootstrapping code? Think that should still be simple enough to do in this PR.